Header header
int32 x
int32 y
float32 radius
bool isDetected
bool isStable
